docs/csharp/programming-guide/classes-and-structs/snippets/local-functions/Program.cs

```diff
@@ -121,5 +123,25 @@ private async Task<int> SecondStep(int index, string name)
             await Task.Delay(100);
             return 9;
         }
+
+        //<YieldReturn>
+        public IEnumerable<string> SequenceToLowercase(IEnumerable<string> input)
+        {
+            if (input.Any())
```

Comment:
```suggestion
            if (!input.Any())
```

This needs to be a negation, otherwise it would throw an exception when there are any items in the `input`.

Reply:
😳 100% right, this was an oversight on my end